test/game_test.rb

```diff
@@ -0,0 +1,90 @@
+require "./test/test_helper"
+require "./lib/game"
+
+class GameTest < Minitest::Test
+
+  def setup
+    game_data_template = {
+      id:           2012030221,
+      season:       20122013,
+      type:         "Postseason",
+      date_time:    "5/16/13",
+      away_team_id: 3,
+      home_team_id: 6,
+      away_goals:   2,
+      home_goals:   3,
+      venue:        "Toyota Stadium",
+      venue_link:   "/api/v1/venues/null"
+    }
+
+    game_data1 = game_data_template.clone
```

Comment:
I'm not familiar with the clone method, but I can see what it is doing here.

Reply:
it's just copying the hash so I'm not modifying the original